hack/update-bazel.sh

```diff
@@ -22,5 +22,8 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 git config http.https://gopkg.in.followRedirects true
 
+# Remove generated files prior to running gazel.
+rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
```

Comment:
If we only use Bazel to build then we can delete this, since Bazel doesn't put the generated file in with the rest of the source. As long as we support both Bazel and make we'll need this.

We could consider adding the ability to gazel to move files to somewhere else and then back after the run, but I don't think it's worth it.